addons/docs/src/mdx/mdx-compiler-plugin.js

```diff
@@ -112,6 +112,7 @@ function genStoryExport(ast, context) {
   }
 
   if (storyAttr) {
+    // NOTE: Maybe we should merge parameters for imported story
```

Comment:
@shilman, what do you think about this? User can write
```jsx
<Story story={importedStoryFromSomewhere} parameters={{ foo: 'bar' }} />
```
And we create yet another story but ignore parameters

Reply:
I'd be up for this change in a separate PR 👍